lute/luau/src/luau.cpp

```diff
@@ -20,6 +27,9 @@
 #include <iterator>
 #include <memory>
 #include <string>
+#include <map>
+#include <optional>
+#include <string>
```

Comment:
why are we re-importing string? can you also alpha-sort each section of includes?